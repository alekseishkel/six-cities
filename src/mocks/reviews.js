const reviews = [
  {
    comment: `Bathed in the nature. Completely unplugged. Unforgettable.`,
    date: `2021-01-13T06:16:14.873Z`,
    id: 1,
    rating: 2,
    user: {id: 11, is_pro: false, name: `Jack`, avatar_url: `https://htmlacademy-react-2.appspot.com/six-cities/static/avatar/2.jpg`}

  },
  {
    comment: `The house is very good, very happy, hygienic and simple living conditions around it are also very good. I hope to have the opportunity to come back. Thank you.`,
    date: `2021-01-13T06:16:14.873Z`,
    id: 2,
    rating: 3,
    user: {id: 12, is_pro: true, name: `Isaac`, avatar_url: `https://htmlacademy-react-2.appspot.com/six-cities/static/avatar/3.jpg`}
  }
];

export default reviews;

