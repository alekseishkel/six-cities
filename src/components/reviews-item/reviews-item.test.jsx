import React from 'react';
import renderer from 'react-test-renderer';

import ReviewsItem from './reviews-item.jsx';

import reviews from '../../mocks/reviews';

it(`ReviewsItem correctly renders after relaunch`, () => {
  const tree = renderer.create(
      <ReviewsItem
        review={reviews[0]}
      />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
