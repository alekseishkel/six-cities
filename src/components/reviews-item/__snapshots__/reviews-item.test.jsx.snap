// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReviewsItem correctly renders after relaunch 1`] = `
<li
  className="reviews__item"
>
  <div
    className="reviews__user user"
  >
    <div
      className="reviews__avatar-wrapper user__avatar-wrapper"
    >
      <img
        alt="Reviews avatar"
        className="reviews__avatar user__avatar"
        height="54"
        src="https://htmlacademy-react-2.appspot.com/six-cities/static/avatar/2.jpg"
        width="54"
      />
    </div>
    <span
      className="reviews__user-name"
    >
      Jack
    </span>
  </div>
  <div
    className="reviews__info"
  >
    <div
      className="reviews__rating rating"
    >
      <div
        className="reviews__stars rating__stars"
      >
        <span
          style={
            Object {
              "width": "40%",
            }
          }
        />
        <span
          className="visually-hidden"
        >
          Rating
        </span>
      </div>
    </div>
    <p
      className="reviews__text"
    >
      Bathed in the nature. Completely unplugged. Unforgettable.
    </p>
    <time
      className="reviews__time"
      dateTime="2021-01-13T06:16:14.873Z"
    >
      январь 2021 г.
    </time>
  </div>
</li>
`;
